test/internet/test-dgram-multicast-ssm-multi-process.js

```diff
@@ -0,0 +1,251 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
New files don't require this boilerplate at the top.

Reply:
Shall I remove it completely?

Reply:
Yes, please.